index.html

```diff
@@ -1771,7 +1782,37 @@ <h2>The <dfn>NFCWatchMode</dfn> enum</h2>
         are dispatched.
       </dd>
     </dl>
-  </section> <!-- NFCWatchMode enum -->
+  </section> <!-- NFCAccept enum -->
+
+  <section><h3>Constructing an <a>NFCReader</a> object</h3>
+    <p>
+      <ol>
+        <li>
+          Let <var>reader</var> be a new <a><code>NFCReader</code></a> object.
+        </li>
+        <li>
+          Let <var>options</var> be first argument to constructor.
+        </li>
+        <li>
+          <a>For each</a> <var>key</var> → <var>value</var> of <var>options</var>:
+          <ul>
+            <li>
+              If the <a>NFCReaderOptions</a>
```

Comment:
This should probably be NFCReader, because otherwise I will copy `signal` as well. Maybe I should just list our the attributes? @anssiko

Reply:
Nit: If you have a normative definition here, then lines 1478-1482 should be a note. But, in WebIDL, attributes are getter/setters, not own properties, so their definition isn't determined by setting the attribute to a value, but saying that, when you read the attribute, you get that value.